### DISCUSSION TITLE
Change keyboard Layout

### DISCUSSION THREAD

**Discussion by josfergrost** [JaKooLit/Hyprland-Dots#476] [Q&A] [answered]
Someone knows how to change keyboard layout or add new layout??

**Comment by dwilliam62** [comment, marked as answer]
> Someone knows how to change keyboard layout or add new layout??

SUPER  + E  Edit User Settings   You will see the keyboard info there to set the layout, etc 

![image](https://github.com/user-attachments/assets/3ce45223-0dae-4436-864c-169caaa8ad42)
